internal/constants/constants.go

```diff
@@ -493,3 +493,6 @@ const IgnoreEnvEnvVarName = "ACTIVESTATE_CLI_IGNORE_ENV"
 
 // ProgressUrlPathName is the trailing path for a project's build progress.
 const BuildProgressUrlPathName = "distributions"
+
+// RuntimeCacheSizeConfigKey is the config key for the runtime cache size.
+const RuntimeCacheSizeConfigKey = "runtime.cache_size"
```

Comment:
```suggestion
const RuntimeCacheSizeConfigKey = "runtime.cache.size"
```

Reply:
I will change this, but note that the AC specified the underscore. We also have `optin.unstable.async_runtime` vs. `optin.unstable.async.runtime`.

Reply:
Bah you're right, thanks for pointing out. I was trying to recall the consistency of the naming but this could really go either way. Generally I would use dot if there might be more settings that follow under a similar category, eg. maybe we would want `runtime.cache.location`, in which cache `cache_size` would be awkward and inconcistent.

I think let's go with the dots since it does seem plausible we might have more here. And starting to wonder if we should just outlaw underscore for config keys, but that's for another day.